autoload/nerve.wad/umapinfo.lmp

```diff
@@ -0,0 +1,102 @@
+map MAP01
+{
+	levelname = "The Earth Base"
+	label = "Level 1"
```

Comment:
Since label and levelname are merely concatenated, don't the label have to contain a separator, e.g. "Level 1: " or "Level 1 - "?

Reply:
It's separated by ":", unless I missed something again :)
![doom00](https://user-images.githubusercontent.com/5077629/119229143-b726b380-bb40-11eb-84a7-55c06d343c68.png)

Reply:
Nope, this time I missed it, you are right. 😉